includes/admin/class-sensei-learners-main.php

```diff
@@ -483,6 +484,7 @@ protected function get_row_data( $item ) {
 					array(
 						'title'            => '<strong><a class="row-title" href="' . esc_url( admin_url( 'user-edit.php?user_id=' . $user_activity->user_id ) ) . '" title="' . esc_attr( $a_title ) . '">' . esc_html( $title ) . '</a></strong>',
 						'date_started'     => get_comment_meta( $user_activity->comment_ID, 'start', true ),
+						'date_completed'   => ( 'complete' === $user_activity->comment_approved ) ? $user_activity->comment_date : '',
```

Comment:
This looks good, although the doc block above the filter needs to be updated to include this new parameter. Actually, it looks like those params are stale so this would be a good opportunity to correct that. It should follow [this format](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).

Reply:
Updated docblock - not sure it's correctly formatted as requested :-S